### DISCUSSION TITLE
Set the redirect for the root / of a custom domain

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#305] [Anything Else]
"can you add an option to set the redirect when someone visits the root / of a custom domain? or at least forward it to the tracked domain instead of an nginx 404?"

(feedback via email)

**Comment by cassidyjames** [comment]
This would alleviate #92 for elementary, at least, as we could build a custom sort of “dashboard” page that lists all our public stats pages, i.e. at elementary.io/stats. And then if someone is poking at the source/requests and wonders what `stats.elementary.io` is, they'd at least get something instead of a 404.

**Comment by ukutaht** [comment]
Thanks @cassidyjames, that makes sense.
